main.py

```diff
@@ -217,11 +222,33 @@ def is_neutral(status):
 @click.option("-v", "--verbose", is_flag=True)
 @click.option("-d", "--dry-run", is_flag=True)
 def cli(dry_run, verbose):
-    bads = run(verbose=verbose, dry_run=dry_run)
-    if bads:
-        click.secho(
-            f"\n{len(bads)} settings have a bad ratio over threshold.", fg="red"
+    use_sentry = False
+    if SENTRY_DSN:
+        sentry_sdk.init(SENTRY_DSN)
+        use_sentry = True
+    elif not DEBUG:
+        click.secho("No SENTRY_DSN set but not in DEBUG mode!", bold=True)
+    try:
+        bads = run(verbose=verbose, dry_run=dry_run)
+    except Exception as exception:
+        sentry_sdk.capture_exception(exception)
```

Comment:
It was not obvious on first read why we had `capture_exception()` and `capture_message()`.
Could you please add a comment that here we capture «unexpected» errors like network, parsing or whatever. And that we report bad status below?

Reply:
I believe I have.